compiler/src/compiler/string_to_rcst.rs

```diff
@@ -300,7 +303,7 @@ mod parse {
             Some((
                 " ",
                 Rcst::Int {
-                    value: 42,
+                    value: BigUint::from(42u8),
```

Comment:
Because `BigUint` implements `From<i64>` (and the same for several types of `uXX`), I think we might be able to write something like this:

```suggestion
                    value: 42.into(),
```

This also applies to the other places where the `BigUint::from(constant)` pattern is used.